client/src/resources/honesty-policy.tsx

```diff
@@ -14,7 +14,7 @@ const HonestyPolicy = () => {
       <p key={5}>{t('settings.honesty.p5')}</p>
       <p key={6}>{t('settings.honesty.p6')}</p>
       <p key={7}>
-        <Trans email={email} i18nKey='settings.honesty.p7'>
```

Comment:
This was removed because `<Trans>` does not support rest props IMO and did not have an email field. Not sure if `email` was being used as a prop on Trans either.